src/wrappers/themis/wasm/wasmthemis.mk

```diff
@@ -28,6 +28,7 @@ $(BIN_PATH)/libthemis.js: LDFLAGS += -s EXPORTED_RUNTIME_METHODS=@$(WASM_RUNTIME
 $(BIN_PATH)/libthemis.js: LDFLAGS += -s ALLOW_TABLE_GROWTH
 $(BIN_PATH)/libthemis.js: LDFLAGS += -s MODULARIZE=1
 $(BIN_PATH)/libthemis.js: LDFLAGS += -s ALLOW_MEMORY_GROWTH=1
+$(BIN_PATH)/libthemis.js: LDFLAGS += -s EXPORTED_FUNCTIONS='["_malloc", "_free"]'
```

Comment:
let's specify them in same way as `-s EXPORTED_RUNTIME_METHODS=@$(WASM_RUNTIME)`